### PR TITLE
Update python integration tests for using the fast miner

### PR DESCRIPTION
**Pull request by velzevur**
Tests used to post premined blocks for populating empty nodes; now they use fast mining instead

### CODE REVIEW

File: py/tests/integration/setup.yaml [hunk 1]
```diff
@@ -41,16 +41,14 @@ tests: # test specific settings
       nodes:
         bob: dev1
         alice: dev2
-      blocks_to_post: 7
-      chain_data: py/tests/integration/data/bchain.txt
```
Comment (lucafavatella) [inline py/tests/integration/setup.yaml]: @velzevur I am double checking the deletion of the pre-mined chain files.

Comment (lucafavatella) [reply py/tests/integration/setup.yaml]: It looks still there.
```
$ find . -name bchain.txt
./py/tests/integration/data/bchain.txt
$ ls -lh ./py/tests/integration/data/bchain.txt
-rw-r--r--  1 luca  staff   7.9K 16 Nov 09:34 ./py/tests/integration/data/bchain.txt
```

I see it is used by [`test_ext_api`](https://github.com/aeternity/epoch/pull/375/files#diff-76bf7aa6bd9ae9d90af0e0d0f5156921R36).

**Comment by lucafavatella** [comment]
I see the [package job on OSX failed](https://travis-ci.org/aeternity/epoch/jobs/303101183):

```
Will mine till block 50
[node1] height=2
[node2] height=2
[node3] height=2

[node1] height=2
[node2] height=2
[node3] height=2

...
```

File: py/tests/integration/setup.yaml (hunk 1 above)
Comment (velzevur) [reply py/tests/integration/setup.yaml]: yes, for this we will need some premined chain; this functionality is covered in the user acceptance tests; I'll fix the API calls in a subsequent PR

**Comment by lucafavatella** [comment]
> I see the package job on OSX failed:

I am attempting to reproduce locally (by building package and then running release script - as CI) but I am not managing to reproduce test failure.

File: py/tests/integration/test_use_cases.py [hunk 2]
```diff
@@ -176,3 +217,22 @@ def node_peer_url(node_name):
     host = common.node_config(node_name)["host"]
     port = common.node_config(node_name)["ports"]["external_api"]
     return "http://" + host + ":" + str(port) + "/"
+
+def make_fast_mining_config(root_dir, file_name):
+    sys_config = os.path.join(root_dir, file_name)
+    f = open(sys_config, "w")
+    # if autostart is not true - there will be no miner
+    conf ='[{aecore, [{autostart, true},' + \
+                    ' {aec_pow_cuckoo, {"lean16", "-t 5", 16}}]}].'
```
Comment (lucafavatella) [inline py/tests/integration/test_use_cases.py]: Does this mean that https://github.com/aeternity/epoch/blob/ccbc61fa6e938661bc0c7d238624f54d4fe41fb0/py/tests/release.py#L20 can be deleted?

**Pull request merged**